examples/quickstarts/helidon-quickstart-se/src/main/docker/Dockerfile.native

```diff
@@ -20,21 +20,21 @@ WORKDIR /helidon
 ADD *.jar target/
 ADD ${libs.classpath.prefix}/*.jar target/
 
-RUN echo "$GRAALVM_HOME/bin/native-image -H:Name=helidon-native -H:IncludeResources='application.yaml|logging.properties' -H:+ReportExceptionStackTraces -classpath target/* ${main.class}"
-RUN $GRAALVM_HOME/bin/native-image \
+RUN echo "${GRAALVM_HOME}/bin/native-image -H:Name=${project.artifactId}-native -H:IncludeResources='application.yaml|logging.properties' -H:+ReportExceptionStackTraces -classpath target/* ${mainClass}"
+RUN ${GRAALVM_HOME}/bin/native-image \
         --static \
-        -H:Name=helidon-native \
+        -H:Name=${project.artifactId}-native \
```

Comment:
This does not have to be project specific. We can use the hardcoded value. That way the generated image is the same for every project (and easier to troubleshoot)

Reply:
As we discussed in today's meeting, this will be removed once I finalize the proper multi-stage docker build, the Dockerfile.native will also use mvn to build the native executable.